logging-k8s-stdout-otlp-json/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM eclipse-temurin:21-jre
 WORKDIR /usr/src/app/
 
 # renovate: datasource=github-releases depName=opentelemetry-java-instrumentation packageName=open-telemetry/opentelemetry-java-instrumentation
-ENV OPENTELEMETRY_JAVA_INSTRUMENTATION_VERSION=v2.12.0
+ENV OPENTELEMETRY_JAVA_INSTRUMENTATION_VERSION=v2.11.0
```

Comment:
I bumped this back a version in order to verify once this is merged that renovate is still finding and updating it